packages/import-bundle/test/import-bundle.test.js

```diff
@@ -19,13 +19,13 @@ function transform1(src) {
     .replace('two foot wide', 'sixty feet wide');
 }
 
-async function testBundle1(t, b1, mode, ew) {
-  const ns1 = await importBundle(b1, { endowments: ew });
+async function testBundle1(t, b1, mode, ew, opt = {}) {
```

Comment:
Style: please avoid abbreviation.
```suggestion
async function testBundle1(t, b1, mode, ew, options = {}) {
```

If you find yourself doing an initialism or an acronym, treat it as a word. We say `XmlHttpRequest` ’round these here parts.